pkg/source/reconciler/streamsource/streamsource.go

```diff
@@ -127,16 +127,18 @@ func (r *Reconciler) updateRedisConfig(ctx context.Context, configMap *corev1.Co
 	redisConfig, err := GetRedisConfig(configMap.Data)
 	if err != nil {
 		logging.FromContext(ctx).Errorw("Error reading Redis configuration", zap.Error(err))
+	} else {
+		// For now just override the previous config.
+		r.numConsumers = redisConfig.NumConsumers
 	}
-	// For now just override the previous config.
-	r.numConsumers = redisConfig.NumConsumers
 }
 
 func (r *Reconciler) updateTLSSecret(ctx context.Context, secret *corev1.Secret) {
 	tlsSecret, err := GetTLSSecret(secret.Data)
 	if err != nil {
 		logging.FromContext(ctx).Errorw("Error reading TLS configuration", zap.Error(err))
+	} else {
```

Comment:
nit: to make the code a bit cleaner here, instead of using an `else`, could you `return` in the `if err != nil` block?